app/controllers/session_controller.rb

```diff
@@ -594,7 +594,7 @@ def destroy
       client_ip: request&.ip,
       user_agent: request&.user_agent,
     }
-    DiscourseEvent.trigger(:before_session_destroy, event_data)
```

Comment:
In Ruby 2, `event_data` is converted to keyword arguments, which are then collected into a hash in `#trigger`, but that will be a different hash.